README.md

```diff
@@ -54,25 +54,28 @@ See the help text for more detailed usage instructions.
 
 ## Semantic Versioning
 
-`dep ensure` a uses an external [semver library](https://github.com/Masterminds/semver) to interpret the version constraints you specify in the manifest. The basic comparison operators are:
+`dep ensure` a uses an external [semver library](https://github.com/Masterminds/semver) to interpret the version constraints you specify in the manifest. The comparison operators are:
 
 * `=`: equal (aliased to no operator)
 * `!=`: not equal
 * `>`: greater than
 * `<`: less than
 * `>=`: greater than or equal to
 * `<=`: less than or equal to
+* `<=`: less than or equal to
+* `-`: literal range. Eg: 1.2.4 - 1.3
```

Comment:
It's not clear from this example whether or not the range is inclusive. Let's change it to `1.2 - 1.4.5 is equivalent to >= 1.2, <= 1.4.5`.